requirements/base.txt

```diff
@@ -2,12 +2,12 @@
 # This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile base.in
+#    pip-compile requirements/base.in
```

Comment:
Looks like you're running `pip-tools` from a different directory, then before. Be consistent so that it doesn't generate pointless changes.

Reply:
Adding a 👍 doesn't count as reverting an unrelated change.

Reply:
> Adding a 👍 doesn't count as reverting an unrelated change.

Yes, it looks like this pull request needs a lot of action with fixing commits.

Reply:
Marking this as resolved.